Makefile

```diff
@@ -1,11 +1,13 @@
 SOURCES = $(wildcard *.go)
+TEST?=./...
```

Comment:
I would add launchdarkly in the path to prevent gotest to try running in a folder without tests :
```
go test ./... -timeout=30s -parallel=4
?       github.com/coveo/terraform-provider-launchdarkly        [no test files]
ok      github.com/coveo/terraform-provider-launchdarkly/launchdarkly   (cached)
```